<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ремонтная мастерская</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabMyOrders">
       <attribute name="title">
        <string>Мои заказы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="searchEdit">
          <property name="placeholderText">
           <string>Поиск по клиенту, статусу, дате...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableMyOrders">
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnNewOrder">
            <property name="text">
             <string>Оформить заказ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnInvoice">
            <property name="text">
             <string>Счёт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExportMyOrders">
            <property name="text">
             <string>Экспорт истории</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAllOrders">
       <attribute name="title">
        <string>Все заказы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="searchEdit_2">
          <property name="placeholderText">
           <string>Поиск по клиенту, статусу, дате...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableAllOrders">
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnChangeStatus">
            <property name="text">
             <string>Изменить статус</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAssignParts">
            <property name="text">
             <string>Назначить запчасти</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnInvoice_2">
            <property name="text">
             <string>Счёт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExportAllOrders">
            <property name="text">
             <string>Экспорт истории</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabParts">
       <attribute name="title">
        <string>Склад запчастей</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableParts">
          <property name="columnCount">
           <number>4</number>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <row/>
          <column>
           <property name="text">
            <string>Артикул</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Наименование</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Остаток</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Цена (₽)</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddPart">
            <property name="text">
             <string>Добавить запчасть</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemovePart">
            <property name="text">
             <string>Удалить запчасть</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExportHistory"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExportHistory">
   <property name="text">
    <string>Экспорт истории заказов...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
